packages/amplify-provider-awscloudformation/src/configuration-manager.js

```diff
@@ -509,18 +541,81 @@ function loadConfigFromPath(context, profilePath) {
   throw new Error(`Invalid config ${profilePath}`);
 }
 
-async function loadConfigurationForEnv(context, env) {
+async function getAdminCredentials(idToken, identityId, region) {
+  const cognitoIdentity = new aws.CognitoIdentity({ region });
```

Comment:
is this the proxy configures `aws`? I think not.

Reply:
Marking as TODO post release.